packages/blockchain-wallet-v4-frontend/src/assets/locales/index.d.ts

```diff
@@ -1721,7 +1721,7 @@ type MessagesType = {
   'scenes.authorizelogin.loading.approved.title': 'Login Approved!'
   'scenes.authorizelogin.loading.rejected.content': 'Please contact our support team if you have any questions or concerns.'
   'scenes.authorizelogin.loading.rejected.title': 'Login Attempt Rejected!'
-  'scenes.authorizelogin.or': 'Or'
+  'scenes.authorizelogin.or': 'or'
```

Comment:
just delete this key add a new generic one named `copy.or`. it might already exist even

Reply:
Done